cmake/README.md

```diff
@@ -26,17 +26,25 @@ TBB_BUILD_APPLE_FRAMEWORKS - Enable the Apple* frameworks instead of dylibs, onl
 
 ### Preparation
 
-To perform an out-of-source build, create a build directory and go there:
+To perform an out-of-source build, create a build directory outside of the oneTBB source directory and go there.
 
 ```bash
 mkdir /tmp/my-build
 cd /tmp/my-build
 ```
 
+Or you may also create a build directory within the oneTBB source directory.
+
+```bash
+mkdir ./build && cd ./build
+```
+
 ### Configure
 
+In the build directory, specify the oneTBB source directory to generate a build system for it.
+
 ```bash
-cmake <options> <repo_root>
+cmake <options> <source_dir>
```

Comment:
It is not correct, because the main CMakeList.txt is in <repo_root> - not in /src.

Reply:
I changed it to source because a repo dir is referred as `<path-to-source>` in `cmake` menu:
```
cmake [options] <path-to-source>
```

But I am happy to restore it back to repo to avoid confusion.